### DISCUSSION TITLE
Final Project Group Member Change

### DISCUSSION THREAD

**Discussion by zhangwy324** [comp426-2023-spring/syllabus#2] [General]
Will we be able to add/change team members for the final project later on in the course? Thanks!

**Comment by jesse-wei** [comment]
I asked Professor Martin this question after class today. His response was yes - send updated information (teammates) to the email sent after e00 completion that says "If any of the information below is incorrect, please reply to this message with corrected information ASAP."

**Comment by jdmar3** [comment]
As @jesse-wei says above, we can make personnel changes, but after next week, this will be difficult because I hope to have everyone set in teams so that you can begin working together on things. So, if you have changes to make, now is the time.

**Reply by jesse-wei** [reply]
Since e00 closed 1-17, will updating the survey responses and the e00 repo (I believe updating the responses generates a new EXAMCODE) within the next week incur any grade penalties?
